router/core/graphql_prehandler.go

```diff
@@ -242,10 +249,17 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 
 				writeOperationError(r, w, requestLogger, err)
 				h.releaseBodyReadBuffer(buf)
+				operationReadSpan.End()
 				return
 			}
 		}
 
+		operationReadSpan.SetAttributes(
+			otel.WgGraphQLRequestSizeBytes.Int(len(body)),
```

Comment:
There is standardized OTEL attribute for request content bytes

```go
semconv17 "go.opentelemetry.io/otel/semconv/v1.17.0"
semconv17.HTTPResponseContentLength()
```

Reply:
This is not exactly accurate I was thinking. Independent of multipart or not, we don't want to know the http content length but really just the size of the GraphQL Request, which is also not exactly the body, because of multipart, but only the "GraphQL Request" part.

Reply:
We already know how much bytes are read from the root span.